src/views/todo/TodoView.vue

```diff
@@ -6,45 +6,47 @@
         <SideNavigationComponent :sideBarStatus="this.sideBarOpen" />
       </div>
       <div class="d-flex flex-fill justify-content-center">
-        <TaskContainerComponent v-if="taskIsActive" />
-        <NewPageComponent v-if="newPageIsActive" />
+        <!-- <TaskContainerComponent v-if="taskIsActive" />
+        <NewPageComponent v-if="newPageIsActive" /> -->
+        <component :is="pages[currentPage]"></component>
       </div>
     </div>
   </div>
 </template>
 
 <script>
-import TopNavigationComponent from './subcomponents/TopNavigationComponent.vue'
-import SideNavigationComponent from './subcomponents/SideNavigationComponent.vue'
-import TaskContainerComponent from './subcomponents/MainComponent.vue'
-import NewPageComponent from './subcomponents/NewPageComponent.vue'
-
-import { mapGetters } from 'vuex'
-
-export default {
-  computed: {
-    ...mapGetters({
-      taskIsActive: 'notes/getTaskIsActiveStatus',
-      newPageIsActive: 'notes/getNewPageIsActiveStatus'
-    })
-  },
-  data() {
-    return {
-      sideBarOpen: false
-    }
-  },
-  components: {
-    TopNavigationComponent,
-    SideNavigationComponent,
-    TaskContainerComponent,
-    NewPageComponent
-  },
-  methods: {
-    toggleSideBarOpen() {
-      this.sideBarOpen = !this.sideBarOpen
+    import TopNavigationComponent from './subcomponents/TopNavigationComponent.vue'
+    import SideNavigationComponent from './subcomponents/SideNavigationComponent.vue'
+    import TaskContainerComponent from './subcomponents/MainComponent.vue'
+    import NewPageComponent from './subcomponents/NewPageComponent.vue'
+    import { mapGetters } from 'vuex'
+    
+    export default {
+        computed: {
+        ...mapGetters({
+            taskIsActive: 'notes/getTaskIsActiveStatus',
+            newPageIsActive: 'notes/getNewPageIsActiveStatus',
+            pages: 'notes/getPages',
+            currentPage: 'notes/getCurrentPage'
+        })
+        },
+        data() {
+        return {
+            sideBarOpen: false
+        }
+        },
+        components: {
+        TopNavigationComponent,
+        SideNavigationComponent,
+        TaskContainerComponent,
+        NewPageComponent
+        },
+        methods: {
+        toggleSideBarOpen() {
+            this.sideBarOpen = !this.sideBarOpen
+        }
+        }
```

Comment:
@joven1export nitpick (meaning it's not required to do), but for readability's sake, please do add indents when its under a specific block 😄 this helps us understand which code is under what scope